INSTALL_GUIDE.md

```diff
@@ -0,0 +1,190 @@
+
+- Git
+- GitHub Account (Optional)
+- MiniConda3
+- Copernicus account (Required to pull Weather files which inform simulated weather)
+- Alternate Text Editors/IDE (Optional) = Alternative software like notepad for opening files and editing
+
+## 2. Git
+
+### What is Git? Why do you need it?
+
+Git is an open-source software used for version control, often utilized by programmers to collaborate and share source code during software development. This step can be bypassed if users opt to directly download the ZIP from GitHub. However, keeping code up to date with the newest versions and updates must be done manually for each file and cannot be semi-automated by Git’s control system.
+
+### Installing Git
+
+To install Git, visit the website (<https://git-scm.com/>).
+Navigate to the downloads tab and select the appropriate operating system for your system.
+
+Choose the correct installer corresponding to your system. To determine this, search "About your PC" in your computer's search bar.
+
+After the download completes, open the installer and proceed by following the provided instructions.
+
+### Accessing Git
+
+For Windows 11, to access the Git terminal window at a specific location, right-click within File Explorer that's opened to the desired location and select "Show more options." This will display additional options.
+
+Select "Git Bash Here." This action will open the Git Bash terminal.
+
+Further steps for utilizing this window will be explained subsequently.
+
+## 3. MiniConda3
+
+### What is MiniConda? Why do you need it?
+
+MiniConda is a software designed to manage various packages (code libraries) for Python. It aids in maintaining and tracking the Python packages necessary for LDAR-Sim.
+
+Users are encouraged to select and employ the Python package managers they are most familiar with, as there are multiple alternatives available, including Pipenv, Docker, etc. Nevertheless, this guide will specifically concentrate on the utilization of MiniConda.
+
+### Installing MiniConda3
+
+Visit the MiniConda download website (<https://docs.conda.io/en/latest/miniconda.html>) and download the appropriate installer for your system.
+
+Execute the installer executable and proceed with the installation of MiniConda.
+
+![alt text](doc-images/image5.png)
+
+## 4. LDAR-Sim
+
+### Downloading LDAR-Sim
+
+To download LDAR-Sim from GitHub, there are two available options:
+
+1. Download using Git
+2. Directly downloading the ZIP.
+
+For the first option, where Git is required, users must have Git installed on their system.
+
+The command git clone can be used to clone and download the repository via the Git Bash window:
```

Comment:
Should we mention here that this will download LDAR-Sim to the folder where the git bash in currently active?